utils/app/app.go

```diff
@@ -58,6 +60,7 @@ func Setup(opts *SetupOpts) (*Flags, error) {
 	var flags Flags
 	kingpin.Flag("grpc.addr", "gRPC listen address").Default(":20201").StringVar(&flags.GRPCAddr)
 	kingpin.Flag("debug.addr", "Debug listen address").Default(":20203").StringVar(&flags.DebugAddr)
+	kingpin.Flag("github.url", "github url to fetch operator manifests from").Default("https://raw.githubusercontent.com").StringVar(&flags.GithubUserContentURL)
```

Comment:
why do we make this flag? if to make it easy to configure path to operator crds, why don't include `/percona/`? or even pass `url template`?

Reply:
The template is a good idea, more flexible. I will implement it.

Reply:
Flag for each operator introduced.